Extension/src/Debugger/configurationProvider.ts

```diff
@@ -93,8 +93,8 @@ class CppConfigurationProvider implements vscode.DebugConfigurationProvider {
 	 * Returns a list of initial debug configurations based on contextual information, e.g. package.json or folder.
 	 */
     async provideDebugConfigurations(folder: vscode.WorkspaceFolder | undefined, token?: vscode.CancellationToken): Promise<vscode.DebugConfiguration[]> {
-        const buildTasks: vscode.Task[] = await getBuildTasks();
-        if (!buildTasks.length) {
+        let buildTasks: vscode.Task[]; // = await getBuildTasks(); // TODO
+        if (!buildTasks || buildTasks.length === 0) {
```

Comment:
Btw, `buildTasks` is undefined as it is never initialized. Technically the `buildTasks.length === 0` check isn't necessary

Reply:
Yeah, I know -- I just kept that code in there for later.